python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/router-bgp.j2

```diff
@@ -557,6 +557,16 @@ ASN Notation: {{ router_bgp.as_notation | arista.avd.default('asplain') }}
 | {{ peer_group.name }} | {{ peer_group.activate | arista.avd.default(false) }} | {{ peer_group.encapsulation | arista.avd.default("default") }} |
 {%             endfor %}
 {%         endif %}
+{%         if router_bgp.address_family_evpn.neighbors is arista.avd.defined %}
+
+##### EVPN Neighbors
+
+| Neighbor | Activate | Encapsulation |
+| -------- | -------- | ------------- |
+{%             for neighbor in router_bgp.address_family_evpn.neighbors | arista.avd.natural_sort('ip_address') %}
+| {{ neighbor.ip_address }} | {{ neighbor.activate | arista.avd.default(false) }} | {{ neighbor.encapsulation | arista.avd.default("default") }} |
```

Comment:
The word "default" is not really helping here. Either set it to `-` like we do elsewhere or set it to `vxlan` which I believe is the EOS default.

Reply:
well I copied this code from peer-groups which uses `default` should I change it here as well?

Reply:
actually default is correct as if we don't comfigure anything it inherits from neighbor default

Reply:
agree to resolve?